pkgs/by-name/pi/pied/package.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  fetchFromGitHub,
+  flutter327,
+  gst_all_1,
+}:
+
+flutter327.buildFlutterApplication rec {
+  pname = "pied";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "Elleo";
+    repo = "pied";
+    rev = "refs/tags/v${version}";
```

```suggestion
    tag = "v${version}";
```
Recently introduced and now preferred.

---

Done